src/devices/cpu/i386/i386.cpp

```diff
@@ -3430,7 +3542,9 @@ void pentium4_device::device_reset()
 	// [ 0:0] FPU on chip
 	// [ 8:8] CMPXCHG8B instruction
 	// [15:15] CMOV and FCMOV
-	m_feature_flags = 0x00008101;       // TODO: enable relevant flags here
+	// [25:25] SSE1
+	// [26:26] SSE2
+	m_feature_flags = 0x0780a1ff;       // TODO: enable relevant flags here
```

Comment:
Think I've already told you once: enabling features that our p2/p3/p4 cores don't even reach is recipe for disaster.  Not even a Voodoo 1 is exercised outside Arcade machines, 8514 needs work and we don't have a functional AGP infrastructure at all, enabling unhandled SSE1/2 will just annoy anybody genuinely interested in testing these cores.